### PR TITLE
Pin firefox version in sandbox playbook

### LINKED ISSUE #14: Firefox version in Beaker-in-a-box is not working properly with selenium
Firefox version included in 7.7 is not working properly.
We should downgrade Firefox in play.

### PR DESCRIPTION
**Pull request by renanrodrigo**
Fix #14

Signed-off-by: Renan Rodrigo <rebarbos@redhat.com>

### CODE REVIEW

File: roles/beaker/sandbox/packages/tasks/main.yml [hunk 1]
```diff
@@ -59,3 +61,32 @@
         - python-glanceclient
         - python-novaclient
         - python-neutronclient
+
+- name: Download Firefox 52 # For selenium compatibility
+  get_url:
+    url: https://ftp.mozilla.org/pub/firefox/releases/52.7.0esr/linux-x86_64/en-US/firefox-52.7.0esr.tar.bz2
+    dest: /tmp/firefox-52.7.0esr.tar.bz2
```
Comment (StykMartin) [inline roles/beaker/sandbox/packages/tasks/main.yml]: Can we introduce a check so we will not download this over and over if we have 52 in beaker sandbox already?

**Review by StykMartin — CHANGES_REQUESTED** [review]

File: roles/beaker/sandbox/packages/tasks/main.yml (hunk 1 above)
Comment (renanrodrigo) [reply roles/beaker/sandbox/packages/tasks/main.yml]: This is the default behavior, as seen in docs: https://docs.ansible.com/ansible/latest/modules/get_url_module.html

Comment (StykMartin) [reply roles/beaker/sandbox/packages/tasks/main.yml]: `/tmp` is the problem then

Comment (renanrodrigo) [reply roles/beaker/sandbox/packages/tasks/main.yml]: Yes. Changed.

**Review by StykMartin — APPROVED** [review]
LGTM

**Pull request merged**